projects/BFDR.Tests/BirthRecord_Should.cs

```diff
@@ -1176,14 +1176,12 @@ public void ParseFatherRaceEthnicityIJEtoJson()
         }
         Assert.Equal(15, b2.FatherRace.Length);
     }
-
     [Fact]
     public void IdentifiersPresent()
     {
-      Assert.Equal("100", FakeBirthRecord.CertificateNumber);
```

Comment:
this is part of the error, Identifier was a placeholder field that we used in early testing. It has been replaced with CertificateNumber